website/docs/feature-flag-tutorials/use-cases/user-management-access-controls.md

```diff
@@ -0,0 +1,172 @@
+
+SCIM takes group synchronization to the next level by providing a standardized protocol for user and group management.
+Through SCIM, you can:
+
+-   [Provision and de-provision users](/reference/scim) (team members) as they are joining or leaving your organization.
+-   Automatically create and delete user groups.
+-   Sync group membership in real-time.
+-   Ensure consistent access across multiple platforms.
+
+Enable SCIM protocol in the **Single sign-on > SCIM** tab to generate a new token for your SCIM client.
+
+![Turn the **SCIM provisioning** toggle on to enable SCIM.](/img/use-case-user-mgmt-scim-protocol.png)
+
+Next, set up [Okta provisioning](/how-to/how-to-setup-provisioning-with-okta) or [Microsoft Entra ID provisioning](/how-to/how-to-setup-provisioning-with-entra) using our how-to guides.
+
+This automation creates an access control system that adapts in real-time to organizational changes. Within minutes of joining, a new team member can be granted precisely the right level of access, while departing employees are immediately locked out of sensitive systems.
+
+## Configure role-based access controls
+
+[Role-based access control](/reference/rbac) makes feature flag management a strategic governance tool. Consider a scenario in which a junior developer should never be able to modify critical feature flags in the authentication system within a banking platform. RBAC makes this granular control seamless. We recommend carefully delegating administrative privileges to users based on the needs of their roles within projects and the organization.
+
+In the Unleash Admin UI, go to **Admin > Roles** to view, create, and manage user roles.
+
+![Manage all user roles in the Unleash Admin UI.](/img/use-case-user-mgmt-root-roles.png)
+
+We have 5 [predefined roles](/reference/rbac#predefined-roles) within our RBAC framework at Unleash.
+
+_Root roles_:
+
+1. Admin
+2. Editor
+3. Viewer
+
+_Project roles_:
+
+1. Owner
+2. Member
+
+Assign users with root roles to configure entire systems. These users can perform any operation within the Unleash platform and change permissions for other users when they belong to a specific user group.
+
+Assign other users to project roles to have domain-specific control within a specific project maintained in Unleash.
+
+Project permissions are separated from root permissions to make it even more targeted regarding what permissions someone can and cannot have for each piece of Unleash. Assign developers with creation and modification rights and viewers who can observe but not change.
+
+For more fine-tuned access controls, create [custom root roles](/how-to/how-to-create-and-assign-custom-root-roles) and [custom project roles](/how-to/how-to-create-and-assign-custom-project-roles), where you can define the privileges and limitations beyond the predefined roles we have built into Unleash.
```

Comment:
Can we give some examples of the types of access RBAC can be used for here?  I think there are something like 50+ things you can define. We could add a screenshot of that.